views/settings.tpl

```diff
@@ -40,7 +40,8 @@
 
     <h4>SSH</h4>
     <hr>
-    SSH allows you to access the command line of this machine remotely. Once your SSH public key has been added, you can connect to this server with "ssh {{username}}@{{hostname}}".<br>
+    To access this machine remotely, add your public SSH key by pasting it in the field below, then click on "Save" at the bottom of the page. If the operation is successful, you'll see your public key appear below the "Add public key" field once tha page reloads. For more details, please refer to <a href="https://github.com/ChimeraOS/chimera/blob/master/README.md">the README file of the Github repository</a>.<br>
```

Comment:
My concern with this is that it is too long of a description. Perhaps we could make this message a tooltip/popup?

Reply:
The rest of this PR is great, so perhaps at the very least, you could split this part out into a separate PR.

Reply:
> Perhaps we could make this message a tooltip/popup?

HTML isn't exactly my thing. This was an easy enough edit even without being able to properly preview the end result. I wouldn't know how to make anything prettier than this.
In principle I don't disagree. But I would argue that a longer description doesn't hurt, while someone else works on a nicer implementation.

> you could split this part out into a separate PR

Pardon my ignorance, but how would I do that? Make a new commit on this branch that skips this file, and then create a new branch just for this file?

Reply:
We could also opt for a middle-of-the-road solution, and go for:


> SSH allows you to access the command line of this machine remotely. Once your SSH public key has been added, you can connect to this server with "ssh {{username}}@{{hostname}}".
> For more details, please refer to <a href="https://github.com/ChimeraOS/chimera/blob/master/README.md">the README file of the Github repository</a>.

Reply:
> We could also opt for a middle-of-the-road solution, and go for:
> 
> > SSH allows you to access the command line of this machine remotely. Once your SSH public key has been added, you can connect to this server with "ssh {{username}}@{{hostname}}".
> > For more details, please refer to [the README file of the Github repository](https://github.com/ChimeraOS/chimera/blob/master/README.md).

Yes, that works.